azure-keyvault/HISTORY.rst

```diff
@@ -2,6 +2,27 @@
 
 Release History
 ===============
+1.0.0b1 (2018-04-10)
```

Comment:
Should this file includes a 1.0.0 now? Even if it's just "release 1.0.0b1 as 1.0.0 unchanged"?